resources/com/google/enterprise/adaptor/TranslationStrings.properties

```diff
@@ -63,5 +63,7 @@ STATUS_JAVA_VERSION=Java Version
 STATUS_JAVA_VERSION_SUPPORTED=Version {0} of Java is supported.
 # The current JVM is not known (i.e. not a standardly-named JVM)
 STATUS_JAVA_VERSION_UNKNOWN=Version {0} of Java is not recognized.  Please ensure it is version {1} (or later).
+# The current JVM is partially unsupported
+STATUS_JAVA_VERSION_PARTIAL=Version {0} of Java is partially supported.  Please upgrade to version {1} (or later).
```

Comment:
This message doesn't communicate the seriousness of the situation. That is, how is admin to judge whether this is serious or trivial.  I think a specific message, and if we can only when we know conditions are met, would be more meaningful and actionable.  Is it possible to detect the condition so as to not scare unnecessarily?

Reply:
Rather than adding yet more code, which I don't have any idea how to write or whether that's possible, what if we just replace the second sentence, "Please upgrade ...", to "Please check the supported Java versions in the Administration Guide for details."? This is in all cases a diagnostic message in the logs and dashboard, not a failure.

Reply:
Why say "partial"?  Why not say that the version of Java has security problem with Diffie–Hellman?